packages/vuetify/src/mixins/menuable/index.ts

```diff
@@ -246,7 +362,7 @@ export default baseMixins.extend<options>().extend({
       }
     },
     checkActivatorFixed () {
-      if (this.attach !== false) return
+      if (this.attach) return
```

Comment:
This is intentional, `''` is falsy but should be treated as true

Reply:
Interesting, in my testing the attach prop defaults to undefined instead of false as well so the results came out wrong, should I change the default value then?

Reply:
Okay so it seems it's basically expecting the Detachable mixin to always be included with the Menuable one, so maybe the Menuable should extend the Detachable mixin?